### DISCUSSION TITLE
TLE Canary:  Minor request to have option to disable warning

### DISCUSSION THREAD

**Discussion by qualiall** [slyh/Cities2-TrafficLightsEnhancement#108] [Ideas]
For TLE Canary mode, would it be possible for you to add a tick option to no longer display the warning message when selecting the tool?  If such a warning needs to be re-enabled for whatever reason after an update, I get it but otherwise I've seen it, read it, understand it.  Don't need to see it each time.  Thanks for consideration of this idea

**Comment by slyh** [comment]
I will add an option to temporarily suppress the warning in the next update. For now, you can switch to alpha, as both alpha and canary are now on the same version.
